data/ui/gametile.ui

```diff
@@ -54,6 +54,46 @@
       </object>
     </child>
   </object>
+  <object class="GtkPopover" id="menu_wine">
+    <property name="can_focus">False</property>
+    <child>
+      <object class="GtkBox">
+        <property name="visible">True</property>
+        <property name="can_focus">False</property>
+        <property name="orientation">vertical</property>
+        <child>
+          <object class="GtkModelButton" id="menu_win_winecfg">
+            <property name="visible">True</property>
+            <property name="can_focus">True</property>
+            <property name="receives_default">True</property>
+            <property name="text" translatable="yes" context="support" comments="Opens the official support webpage">Winecfg</property>
```

Comment:
Do change the translation information in glade for these buttons

Reply:
I also think the button should say something less specific which makes some sense to people not familiar with wine. Using "Settings" or "Configuration" would probably work better.